### PR TITLE
[plugin.video.rivedila7] 6.0.1

### PR DESCRIPTION
**Pull request by luivit**
### Description
plugin's code rework
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.rivedila7/.gitignore [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Byte-compiled / optimized / DLL files
```
Comment (enen92) [inline plugin.video.rivedila7/.gitignore]: drop from the submission

File: plugin.video.rivedila7/resources/lib/plugin.py [hunk 2]
```diff
@@ -4,948 +4,865 @@
...
     if link_video == '':
-        xbmc.log('NO VIDEO LINK',xbmc.LOGINFO)
-        if xbmcgui.Dialog().ok(addon.getAddonInfo('name'), language(32005)):
-            exit()
-    else:
-        listitem.setProperty('inputstream','inputstream.adaptive')
-        listitem.setProperty('inputstream.adaptive.manifest_type','hls')
-        listitem.setPath(link_video)
-        xbmcplugin.setResolvedUrl(handle, True, listitem)
+        xbmc.log('NO VIDEO LINK', xbmc.LOGINFO)
```
Comment (enen92) [inline plugin.video.rivedila7/resources/lib/plugin.py]: can you add some sort of plugin identifier to this log line? Anyone checking it on the log won't notice where it's coming from

**Review by enen92 — CHANGES_REQUESTED** [review]
see comments, the .gitignore has to go

File: plugin.video.rivedila7/.gitignore (hunk 1 above)
Comment (Testato) [reply plugin.video.rivedila7/.gitignore]: you mean that we need remove completely the.gitignore file ? or only this first line ?

Comment (enen92) [reply plugin.video.rivedila7/.gitignore]: All of it

Comment (Testato) [reply plugin.video.rivedila7/.gitignore]: done

File: plugin.video.rivedila7/resources/lib/plugin.py (hunk 2 above)
Comment (Testato) [reply plugin.video.rivedila7/resources/lib/plugin.py]: done

**Review by enen92 — APPROVED** [review]

**Pull request merged**